File: seed/csharp-sdk/exhaustive/no-root-namespace-for-core-classes/src/SeedApi.DynamicSnippets/Example64.cs
using SeedApi;
using SeedApi.Core;
using SeedApi.Endpoints;

namespace Usage;

public class Example64
{
    public async Task Do() {
        var client = new SeedApiClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.Endpoints.Params.GetWithPathAsync(
            new GetWithPathParamsRequest {
                Param = "param"
            }
        );
    }

}

File: seed/java-sdk/exhaustive/custom-client-class-name/src/main/java/com/snippets/Example64.java
package com.snippets;

import com.seed.api.Best;
import com.seed.api.resources.endpoints.params.requests.GetWithPathParamsRequest;

public class Example64 {
    public static void main(String[] args) {
        Best client =
                Best.builder().token("<token>").url("https://api.fern.com").build();

        client.endpoints()
                .params()
                .getWithPath(GetWithPathParamsRequest.builder().param("param").build());
    }
}
